### PR TITLE
Handle the files being imported into the application for historic data

### PR DESCRIPTION
**Pull request by todd-dembrey**

### CODE REVIEW

File: opentech/apply/funds/management/commands/migration_base.py [hunk 1]
```diff
@@ -260,3 +283,17 @@ def get_workflow_state(self, node):
...
         return value.replace('\r\n', '<br>\n')
+
+    def process_file(self, value):
+        if isinstance(value, dict):
+            value = [value]
+
+        files = []
+
+        for file_data in value:
+            file_path = 'files/private/application_upload/' + file_data['filename']
```
Comment (frjo) [inline opentech/apply/funds/management/commands/migration_base.py]: Application uploads are in a number of different directories unfortunately. The code need to look in file_data['uri'] and parse it for path and filename.

File: opentech/apply/funds/management/commands/migration_base.py [hunk 2]
```diff
@@ -260,3 +283,17 @@ def get_workflow_state(self, node):
...
+            saved_file = migration_storage.open(file_path)
+            saved_file.name = file_data['filename']
```
Comment (frjo) [inline opentech/apply/funds/management/commands/migration_base.py]: Drupal store the original file name in "filename" but the actual file can be renamed so not to overwrite files, Drupal adds "_0", "_1" etc. to the name. This also happens when you simply upload a new version with the same filename so there are a few of them.

Comment (todd-dembrey) [reply opentech/apply/funds/management/commands/migration_base.py]: We are doing the same in django. Fixed in 050f1de

File: opentech/apply/funds/management/commands/migration_base.py (hunk 1 above)
Comment (todd-dembrey) [reply opentech/apply/funds/management/commands/migration_base.py]: Fixed in 050f1de

**Pull request merged**